generatekey.php

```diff
@@ -9,3 +9,7 @@ function get_key($bit_length = 128){
     }
     return null;
 }
+//display or return key.
```

Comment:
100% optional, this is just to demonstrate the functionality, however the repo is just a simple function for people to use.